src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -8663,34 +8663,35 @@ namespace HVACVariableRefrigerantFlow {
         }
 
         // Set inlet air mass flow rate based on PLR and compressor on/off air flow rates
-        SetAverageAirFlow(VRFTUNum, 1.0, temp);
+        //SetAverageAirFlow(VRFTUNum, 1.0, temp);
         VRFInletNode = this->VRFTUInletNodeNum;
         T_TU_in = Node(VRFInletNode).Temp;
         W_TU_in = Node(VRFInletNode).HumRat;
-        T_coil_in = T_TU_in;
-        W_coil_in = W_TU_in;
 
         // Simulation the OAMixer if there is any
         if (this->OAMixerUsed) {
-            SimOAMixer(this->OAMixerName, false, this->OAMixerIndex);
+            //SimOAMixer(this->OAMixerName, false, this->OAMixerIndex);
 
             OAMixerNum = UtilityRoutines::FindItemInList(this->OAMixerName, OAMixer);
             OAMixNode = OAMixer(OAMixerNum).MixNode;
             T_coil_in = Node(OAMixNode).Temp;
             W_coil_in = Node(OAMixNode).HumRat;
+        } else {
+            T_coil_in = T_TU_in;
+            W_coil_in = W_TU_in;
         }
         // Simulate the blow-through fan if there is any
-        if (this->FanPlace == BlowThru) {
-            if (this->fanType_Num == DataHVACGlobals::FanType_SystemModelObject) {
-                HVACFan::fanObjs[this->FanIndex]->simulate(1.0 / temp, ZoneCompTurnFansOn, ZoneCompTurnFansOff, _);
-                FanOutletNode = HVACFan::fanObjs[this->FanIndex]->outletNodeNum;
-            } else {
-                Fans::SimulateFanComponents("", false, this->FanIndex, FanSpeedRatio, ZoneCompTurnFansOn, ZoneCompTurnFansOff);
-                FanOutletNode = Fans::Fan(this->FanIndex).OutletNodeNum;
-            }
-            T_coil_in = Node(FanOutletNode).Temp;
-            W_coil_in = Node(FanOutletNode).HumRat;
-        }
+        //if (this->FanPlace == BlowThru) {
```

Comment:
Fan has already been simulated. If the intent here is to simulated at full flow then why weren't the coils simulated again?